### ISSUE TITLE
Tabbing through poi details opens and closes the collapsibles

### ISSUE THREAD

**Issue by steffenkleinle** [digitalfabrik/integreat-app#2572]
### Describe the Bug

If I am tabbing through the poi details, the collapsibles collapse and open seemingly randomly.

### Steps to Reproduce

1. Go to https://webnext.integreat.app/testumgebung/de/locations/ballonfabrik
2. Tab until reaching "Kontaktdetails"
3. See collapsing
4. Tab some more forward/backwards to see more collapsing/opening

### Expected Behavior

The collapsibles should not open/close on tab. Instead, they should only open/close if another key is pressed.
Also, the whole button/collapsible header should have an accessibility label, not just the icon.

### Actual Behavior

The collapsibles open/close on tab.

### Additional Information

None.

**Comment by steffenkleinle** [comment]
Already fixed in #2574.
